examples/tutorials/tutorial_BET_Disk.py

```diff
@@ -96,14 +85,14 @@
         ),
         operating_condition=fl.AerospaceCondition.from_mach(
             mach=0.182,
-            alpha=5 * u.deg,
-            beta=0 * u.deg,
+            alpha=5 * fl.u.deg,
+            beta=0 * fl.u.deg,
             thermal_state=fl.ThermalState(
                 temperature=288.15,
                 material=fl.Air(
                     dynamic_viscosity=fl.Sutherland(
                         reference_temperature=288.15,
-                        reference_viscosity=4.29166e-08 * u.flow360_viscosity_unit,
+                        reference_viscosity=4.29166e-08 * fl.u.flow360_viscosity_unit,
                         effective_temperature=110.4,
                     )
                 ),
```

Comment:
We need to update this with the `operating_condition_from_mach_muref` function, right?

Reply:
I used AerospaceCondition.from_mach instead as it is closer to how it should be defined